### PR TITLE
Profil meetings

### PR DESCRIPTION
**Pull request by HeloiseFerla**

### CODE REVIEW

**Review by Quentin-Macquart — DISMISSED** [review]

File: src/pages/Profile/Meetings/FirstSummary/index.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import { useSelector } from 'react-redux';
+import Participants from './Participants';
+
+export default function FirstSummary() {
+  const [meetings, setMeetings] = useState([]);
+  const [active, setActive] = useState(false);
+  const user = useSelector((state) => state.user);
+
+  const handleToggle = (e) => {
+    console.log(e);
```
Comment (Hacene010) [inline src/pages/Profile/Meetings/FirstSummary/index.jsx]: no console.log !!

File: src/pages/Profile/Meetings/SecondSummary/index.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import { useSelector } from 'react-redux';
+import Participants from './Participants';
+
+export default function SecondSummary() {
+  const [meetings, setMeetings] = useState([]);
+  const [active, setActive] = useState(false);
+  const user = useSelector((state) => state.user);
+
+  const handleToggle = (e) => {
+    console.log(e);
```
Comment (Hacene010) [inline src/pages/Profile/Meetings/SecondSummary/index.jsx]: nope !!

File: src/pages/Profile/Meetings/style.jsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      .participant {
+        margin-left: 10px;
+      }
+    }
+  }
+  .participants {
+    justify-content: center;
+    margin: 45px 0px;
+    height: 30px;
+
```
Comment (Hacene010) [inline src/pages/Profile/Meetings/style.jsx]: empty line

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Some changes to make plz.

**Review by Quentin-Macquart — DISMISSED** [review]

**Review by Hacene010 — APPROVED** [review]

**Review by Quentin-Macquart — APPROVED** [review]

**Pull request merged**